src/index.ts

```diff
@@ -623,6 +623,7 @@ const formats: IFormat[] = [
   hexChecksumChain('ETH', 60),
   hexChecksumChain('ETC', 61),
   getConfig('ICX', 74, icxAddressEncoder, icxAddressDecoder),
+  bitcoinBase58Chain('XVG',77, [[0x1E]], [[0x21]]),
```

Comment:
Can you specify where you found the version number of the [0x1E]], [[0x21]?

Reply:
Found it at https://pkg.go.dev/github.com/assetsadapterstore/verge-adapter/verge#pkg-variables